horace_core/sqw/file_io/@horace_binfile_interface/private/parse_get_sqw_args_.m

```diff
@@ -22,7 +26,8 @@
     'nopix', ...
     'legacy', ...
     'sqw_struc',...
-    'file_backed'
+    'file_backed',...
+    'force_pix_location'...
```

Comment:
`force_pix_location` doesn't give any indication about what this does to me.

Reply:
well, do other names indicate?

Reply:
`nopix` presumably gives me no pixels
`legacy` is presumably some legacy compatibility feature
`sqw_struc` presumably returns a structure rather than an sqw object
`file_backed` makes the pixels file backed.

Reply:
I have added the detailed description of the purpose of this key. Its difficult to make short abbreviation on what its intedned to do.